Gopkg.toml

```diff
@@ -53,6 +53,10 @@ required = [
   name = "k8s.io/client-go"
   branch = "release-6.0"
 
+[[constraint]]
+  name = "k8s.io/kubernetes"
+  branch = "release-1.9"
```

Comment:
afaik we decided that for now cluster api would be released independent of k8s releases. does this tie us to a specific k8s version now?

Reply:
Maybe, I am not sure. I think it depends on which kubernetes library we use and if they differ in different versions. 

I am using the same version as other k8s.io repositories. For example, k8s.io/apiserver etc are all using release-1.9. Later when we bump versions, I think we should bump them together.